docs/fundamentals/systems/core/SlotMixin.md

```diff
@@ -14,7 +14,7 @@ The need for this can occur in the following situations:
 
 The help text here needs to be connected to the input element that may live in shadow dom. The input needs to have `aria-describedby="help-text-id".`
 
-2. An interplay of multiple nested web components. For instance:
+- An interplay of multiple nested web components. For instance:
 
 ```html
 <my-fieldset>
```

Comment:
Shouldn;t <div> here have a slot attribute, eg <div slot="msg">Group error message</div> ?

Reply:
It should be clear that this element lives in light dom. Whether this will have a slot or not is dependent on the implementation detail of my-fieldset (does it have a shadow root? If so, are we dealing with a default or named slot?), which is unknown.

In this example I tried to focus on the part about connecting light dom of a child (an input with aria-describedby="msg") with an element of its parent.
I guess we could make this example clearer like this:

```html
<my-fieldset>
  <my-input><input aria-describedby="parent-msg"></my-input>
  <my-input><input aria-describedby="parent-msg"></my-input>
  <div id="parent-msg">Group errror message</div>
</my-fieldset>
```